src/dataflow/src/render/sources.rs

```diff
@@ -184,7 +184,7 @@ where
 
                     (collection, capability)
                 } else if let ExternalSourceConnector::Postgres(_pg_connector) = connector {
-                    unimplemented!("Postgres sources are not supported yet");
+                    unreachable!("rendering Postgres source");
```

Comment:
I've pulled this failure up the stack to `plan_create_source`. Currently, not failing at that point (or sequencing) means that a non-functional source will be inserted into the user's catalog, preventing us from testing.